test/test_validation.jl

```diff
@@ -50,6 +50,8 @@ end
     @test isempty(sys_no_runchecks.data.validation_descriptors)
 end
 
+#= Disable due to strange implemenation. Needs Review.
+Econ fields do not exist for a while already.
```

Comment:
These two tests are testing IS.get_field_descriptor with local data (`data` and `config`).  They are not using actual PSY structs.